backend/lcfs/tests/other_uses/test_other_uses_view.py

```diff
@@ -136,9 +136,27 @@ async def test_save_other_uses_row_create(
     ) as mock_validate_organization_access:
         set_mock_user(fastapi_app, [RoleEnum.SUPPLIER])
         url = fastapi_app.url_path_for("save_other_uses_row")
-        payload = create_mock_schema({}).model_dump()
+        payload = {
```

Comment:
Why are you moving away from the create_mock_schema helper?

Reply:
Thank you!